less/components/sidebar.less

```diff
@@ -19,7 +19,7 @@ html.no-js .sidebar {
   &.teach-like-mozilla .sidebar {
     .sidebarColorizer(@tealSidebarColor);
   }
-  &.clubs .sidebar {
+  &.clubs .sidebar, &.clubs-curriculum-page .sidebar {
```

Comment:
We need to make a new variable for this colour.

To me it looks kinda like a `@radicalPumpkinOrangeSidebarColor`